src/actions/Fetch.js

```diff
@@ -33,6 +34,9 @@ export class Fetch extends BaseAction {
     // Extend fetch options
     options = Object.assign({}, this.defaultOptions, options)
 
+    // Make sure params is an immutable map
+    params = !params || params && Immutable.Iterable.isIterable(params) ? params : Immutable.fromJS(params)
```

Comment:
Technically `Immutable.fromJS(null)` returns `null` so we could simplify this to:
```
params = Immutable.Iterable.isIterable(params) ? params : Immutable.fromJS(params)
```